### PR TITLE
docs; Better description View Identifier

### PR DESCRIPTION
**Pull request by doctrino**
## Description
Please describe the change you have made.

## Checklist:
- [ ] Tests added/updated.
- [ ] Documentation updated. Documentation is generated from docstrings - these must be updated according to your change.
  If a new method has been added it should be referenced in [cognite.rst](https://github.com/cognitedata/cognite-sdk-python/blob/master/docs/source/cognite.rst) in order to generate docs based on its docstring.
- [ ] Changelog updated in [CHANGELOG.md](https://github.com/cognitedata/cognite-sdk-python/blob/master/CHANGELOG.md).
- [ ] Version bumped. If triggering a new release is desired, bump the version number in [_version.py](https://github.com/cognitedata/cognite-sdk-python/blob/master/cognite/client/_version.py) and [pyproject.toml](https://github.com/cognitedata/cognite-sdk-python/blob/master/pyproject.toml) per [semantic versioning](https://semver.org/).

### CODE REVIEW

File: cognite/client/_api/data_modeling/views.py [hunk 1]
```diff
@@ -99,7 +99,10 @@ def retrieve(
         """`Retrieve a single view by id. <https://developer.cognite.com/api#tag/Views/operation/byExternalIdsViews>`_
 
         Args:
-            ids (ViewIdentifier | Sequence[ViewIdentifier]): View identifier(s)
+            ids (ViewIdentifier | Sequence[ViewIdentifier]): The view identifier(s). This can be given as a tuple of
+                strings or a ViewId object. For example, ("my_space", "my_view"), ("my_space", "my_view", "my_version"),
+                or ViewId("my_space", "my_view", "my_version"). Note version is optinal, if not provided, all versions
```
Comment (doctrino) [inline cognite/client/_api/data_modeling/views.py:104]:
```suggestion
                or ViewId("my_space", "my_view", "my_version"). Note version is optional, if not provided, all versions
```

File: cognite/client/_api/data_modeling/views.py [hunk 2]
```diff
@@ -99,7 +99,10 @@ def retrieve(
         """`Retrieve a single view by id. <https://developer.cognite.com/api#tag/Views/operation/byExternalIdsViews>`_
 
         Args:
-            ids (ViewIdentifier | Sequence[ViewIdentifier]): View identifier(s)
+            ids (ViewIdentifier | Sequence[ViewIdentifier]): The view identifier(s). This can be given as a tuple of
+                strings or a ViewId object. For example, ("my_space", "my_view"), ("my_space", "my_view", "my_version"),
+                or ViewId("my_space", "my_view", "my_version"). Note version is optional, if not provided, all versions
```
Comment (doctrino) [inline cognite/client/_api/data_modeling/views.py:104]:
```suggestion
                or ViewId("my_space", "my_view", "my_version"). Note that version is optional, if not provided, all versions
```

File: cognite/client/_api/data_modeling/views.py [hunk 3]
```diff
@@ -99,7 +99,10 @@ def retrieve(
         """`Retrieve a single view by id. <https://developer.cognite.com/api#tag/Views/operation/byExternalIdsViews>`_
 
         Args:
-            ids (ViewIdentifier | Sequence[ViewIdentifier]): View identifier(s)
+            ids (ViewIdentifier | Sequence[ViewIdentifier]): The view identifier(s). This can be given as a tuple of
+                strings or a ViewId object. For example, ("my_space", "my_view"), ("my_space", "my_view", "my_version"),
+                or ViewId("my_space", "my_view", "my_version"). Note that version is optional, if not provided, all versions
```
Comment (haakonvt) [inline cognite/client/_api/data_modeling/views.py:104]: Is it better to direct the reader to the example section? (And possibly add more examples?)

Comment (doctrino) [reply cognite/client/_api/data_modeling/views.py:104]: Why not both?

Comment (haakonvt) [reply cognite/client/_api/data_modeling/views.py:104]: Both is better ofc! 😉

**Review by haakonvt — APPROVED** [review]

**Pull request merged**